tests/integration_tests/test_mxnet.py

```diff
@@ -0,0 +1,50 @@
+import mxnet as mx
+import numpy as np
+
+import optuna
+from optuna.integration.mxnet import MxnetPruningCallback
+from optuna.testing.integration import DeterministicPruner
+
+
+def test_mxnet_pruning_callback():
```

Comment:
Could you add test cases for covering [mxnet.py#L51](https://codecov.io/gh/Rishav1/optuna/src/2594001adaca5c8399642175896d26c91677403f/optuna/integration/mxnet.py#L51) and [mxnet.py#L55](https://codecov.io/gh/Rishav1/optuna/src/2594001adaca5c8399642175896d26c91677403f/optuna/integration/mxnet.py#L55)?

Reply:
Sorry for late response. I was busy with gsoc proposal submission. I have added more test coverage.

Reply:
Thank you!